ts/BUILD.typescript

```diff
@@ -70,9 +70,9 @@ npm_package(
 npm_link_package(
     name = "node_modules/@bazel/worker",
     src = "npm_at_bazel_worker",
-    deps = [
-        ":node_modules/google-protobuf"
-    ],
+    deps = {
+        ":node_modules/google-protobuf/store/3.20.1": "",
```

Comment:
I totally tried that...

Reply:
Hah. This was my first try as well and is wrong. It should actually be /0.0.0 in this case since its linking there with the vanilla npm_link_package as if it was a 1p dep and not the generated one from the __links repo